### PR TITLE
chore: add changes from from 2.0 release

### PR DESCRIPTION
**Pull request by lifeiscontent**
- add stream function from 2.0 remix release
- minor cleanup from util function
- add tsconfig.json changes from 2.0 remix release

### CODE REVIEW

File: packages/vercel-remix/entry.server.ts [hunk 1]
```diff
@@ -31,14 +32,16 @@ function serveTheBots(
       onAllReady() {
         responseHeaders.set("Content-Type", "text/html");
         let body = new PassThrough();
-        pipe(body);
```
Comment (lifeiscontent) [inline packages/vercel-remix/entry.server.ts:34]: not sure if the positioning of this pipe was important, but at least in the remix 2.0 example projects its after the resolve so thats why I moved it

File: packages/vercel-remix/tsconfig.json [hunk 2]
```diff
@@ -2,12 +2,12 @@
   "include": ["**/*.ts"],
```
Comment (lifeiscontent) [inline packages/vercel-remix/tsconfig.json:2]: these changes are from: https://github.com/remix-run/remix/blob/main/packages/remix-node/tsconfig.json

File: packages/vercel-remix/sessions.ts [hunk 3]
```diff
@@ -65,7 +65,7 @@ export const createKvSessionStorageFactory = (
         }
       },
       async readData(id) {
-        return (await kv.get(id)) || null;
+        return (await kv.get(id)) ?? null;
```
Comment (lifeiscontent) [inline packages/vercel-remix/sessions.ts:68]: I'm not 100% sure if `kv.get(id)` returns anything other than strings, but if it does, this change would not drop falsy values unintentionally

**Comment by lifeiscontent** [comment]
@TooTallNate is there a way I can test these changes on vercel?

**Comment by Klingefjord** [comment]
Not sure how to try this out, but sounds like it might solve this issue?

https://github.com/vercel/remix/issues/60

**Review by TooTallNate — APPROVED** [review]

**Comment by TooTallNate** [comment]
@lifeiscontent For future reference, I have added instructions on how to test changes in this repository here: https://github.com/vercel/remix/wiki/Testing-on-Vercel

**Comment by TooTallNate** [comment]
Thank you!

**Pull request merged**